全然commentできんので仕方なくgithub.comきます...
鯖にアクセス->adminでアクセス試行、のLog:
```
frontend_1  | ℹ ｢hot｣: WebSocket Client Connected
backend_1   | Traceback (most recent call last):
backend_1   |   File "/code/app.py", line 7, in <module>
backend_1   |     'SQLALCHEMY_DATABASE_URI': 'mysql+mysqlconnector://root:password@mysql/db',
backend_1   |   File "/code/api/app.py", line 25, in create_app
backend_1   |     clear_data(db)
backend_1   |   File "/code/api/app.py", line 34, in clear_data
backend_1   |     db.session.execute(table.delete())
backend_1   |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/sqlalchemy/orm/scoping.py", line 153, in do
backend_1   |     return getattr(self.registry(), name)(*args, **kwargs)
backend_1   |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 1176, in execute
backend_1   |     bind, close_with_result=True).execute(clause, params or {})
backend_1   |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 948, in execute
backend_1   |     return meth(self, multiparams, params)
backend_1   |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/sqlalchemy/sql/elements.py", line 269, in _execute_on_connection
backend_1   |     return connection._execute_clauseelement(self, multiparams, params)
backend_1   |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1060, in _execute_clauseelement
backend_1   |     compiled_sql, distilled_params
backend_1   |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1200, in _execute_context
backend_1   |     context)
backend_1   |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1413, in _handle_dbapi_exception
backend_1   |     exc_info
backend_1   |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 203, in raise_from_cause
backend_1   |     reraise(type(exception), exception, tb=exc_tb, cause=cause)
backend_1   |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 186, in reraise
backend_1   |     raise value.with_traceback(tb)
backend_1   |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1193, in _execute_context
backend_1   |     context)
backend_1   |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 508, in do_execute
backend_1   |     cursor.execute(statement, parameters)
backend_1   |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/mysql/connector/cursor_cext.py", line 264, in execute
backend_1   |     raw_as_string=self._raw_as_string)
backend_1   |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/mysql/connector/connection_cext.py", line 380, in cmd_query
backend_1   |     sqlstate=exc.sqlstate)
backend_1   | sqlalchemy.exc.ProgrammingError: (mysql.connector.errors.ProgrammingError) 1146 (42S02): Table 'db.application' doesn't exist [SQL: 'DELETE FROM application'] (Background on this error at: http://sqlalche.me/e/f405)
backend_1   | [2018-06-13 09:58:48,779] WARNING in app: Regenerating test data for development (because FLASK_ENV == development)
backend_1   | 172.18.0.1 - - [13/Jun/2018 09:58:48] "OPTIONS /auth/ HTTP/1.1" 500 -
```

尚、当方`macOS High Sierra 10.13.5`、`chrome  67.0.3396.79`です

--- a/mysql-confd/config.cnf
+++ b/mysql-confd/config.cnf
@@ -1,3 +1,2 @@
 [mysqld]
 character-set-server=utf8
-sql_mode=NO_ENGINE_SUBSTITUTION

